src/components/Card.tsx

```diff
@@ -22,8 +23,28 @@ export const Card = ({ title, author, onPass }: Card) => {
   const topic: PartialKeyword = {
     keyword: title,
     name: author,
+    hashtags: related_topics
   };
 
+  const fetchRelatedTopics = (related_topics: string[]) => {
+      /*
+        related_topics: The list of hashtags - only the words are provided and is already trimmed
+
+        returns: JSX expression for hashtags
+      */
+      const hashtags = related_topics.map(
+        related_topic => (
+          `# ${related_topic}　　`
+        )
+      );
+      return (
+        <div id="hash">
+          <h4>{hashtags}</h4>
+        </div>
+      )
+  };
+  const hashtags = fetchRelatedTopics(related_topics);
+
```

Comment:
- camelCase
- make as Component when returning JSX element.

e.g. 
define it outside of Card component.
```typescript
const FetchRelatedTopics = (...) => {...};
```
and use it like:
```typescript
<FetchRelatedTopics relatedTopics={...} />
```
rather than `{hashtags}`

Reply:
해시태그가 아예 밖으로 나가는 거는 조금 안 예쁠 것 같고 해시태그를 component로 만든다고 하더라도 Card 내에서 사용해야 할 것 같습니다!

Reply:
fecthedRelatedTopics were moved to Description.tsx
Made Description component